client/checkout/woopay/email-input-iframe.js

```diff
@@ -617,13 +616,14 @@ export const handleWooPayEmailInput = async (
 	} );
 
 	if ( ! customerClickedBackButton ) {
-		const paymentMethods = await select(
-			'wc/store/payment'
-		).getAvailablePaymentMethods();
-
-		const hasWCPayPaymentMethod = paymentMethods.hasOwnProperty(
-			'woocommerce_payments'
+		const hasWcPayElementOnBlocks = document.getElementById(
```

Comment:
This seems to have broken WooPay automatic redirection on short code checkout page. https://github.com/Automattic/woocommerce-payments/issues/8398. The selector doesn't exist in short code checkout.

Reply:
I'm working on a fix 👍